test/test_graph.py

```diff
@@ -204,5 +212,33 @@ def test_multiprocessing_reading_service(self) -> None:
         self.assertEqual(d1, d2)
 
 
+class TestGraphVisualization(expecttest.TestCase):
+    @unittest.skipIf(not HAS_GRAPHVIZ, "Package `graphviz` is required to test graph visualization functionalities.")
```

Comment:
@ejguan Do I need to make changes to CI to make sure the package is installed?

Reply:
No need. You can just add this package to https://github.com/pytorch/data/blob/main/test/requirements.txt